skins/index.js

```diff
@@ -45,6 +45,7 @@ import calciumcrew from "./calciumcrew.scss";
 import capy from "./capy.scss";
 import cargo from "./cargo.scss";
 import carrotswap from "./carrotswap.scss";
+import castelloCommunity from "./castello-community.scss"
```

Comment:
You are missing a semicolon at the end of the line.
```suggestion
import castelloCommunity from "./castello-community.scss";
```